src/core/components/PaginatorLink/index.js

```diff
@@ -40,10 +40,14 @@ export default class PaginatorLink extends React.Component {
     }
 
     if (currentPage === page || page < 1 || page > pageCount) {
+      const classNames = makeClassName('Paginate-item', className, {
+        'Paginate-item--current-page': currentPage === page,
+      });
+
       return (
         <Button
-          buttonType="light"
-          className={makeClassName('Paginate-item', className)}
+          buttonType="cancel"
```

Comment:
I know we're using this elsewhere, but it just occurred to me that it's odd we're using a `buttonType` of `cancel` when we're not actually creating a cancel button. Should we have a new type that's more about appearance and not function?

Reply:
Yes, we could rename that in a follow-up since this prop is not used as the html button type anyway.

Reply:
https://github.com/mozilla/addons-frontend/issues/5317